controllers/backstage_controller.go

```diff
@@ -293,6 +298,13 @@ func (r *BackstageReconciler) labels(meta *v1.ObjectMeta, backstage bs.Backstage
 // SetupWithManager sets up the controller with the Manager.
 func (r *BackstageReconciler) SetupWithManager(mgr ctrl.Manager, log logr.Logger) error {
 
+	clientset, err := kubernetes.NewForConfig(mgr.GetConfig())
```

Comment:
Why do we need it?

Reply:
Good catch. Since the PR is already merged, I submitted a follow up PR to cleanup:
https://github.com/janus-idp/operator/pull/177